deployments/liqo/templates/auth-service-service.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    run: auth-service
+  name: auth-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  {{ if .Values.authService.ingress.enable }}
+  type: ClusterIP
+  {{ else }}
+  type: NodePort
+  {{ end }}
+  selector:
+    run: auth-service
+  ports:
+    - name: https
+      protocol: TCP
+      {{ if .Values.authService.ingress.enable }}
+      port: 4999
+      targetPort: 4999
+      {{ else }}
+      port: 442
+      targetPort: 442
```

Comment:
Shouldn't those be 5000 and 443 (at least those are the ports specified in the deployment). Also in the ingress the port is wrong